src/app/contexts/EventTrackingContext/index.tsx

```diff
@@ -21,11 +19,26 @@ import {
   LIVE_PAGE,
   MEDIA_ARTICLE_PAGE,
 } from '#app/routes/utils/pageTypes';
+import { PageTypes, Platforms } from '#app/models/types/global';
 import { ServiceContext } from '../ServiceContext';
 
-export const EventTrackingContext = createContext({});
+type EventTrackingContextProps =
+  | {
+      campaignID: string;
+      pageIdentifier: string;
+      platform: Platforms;
+      producerId: string;
+      statsDestination: string;
+    }
+  | Record<string, never>;
 
-const getCampaignID = pageType => {
+export const EventTrackingContext = createContext<EventTrackingContextProps>(
+  {} as EventTrackingContextProps,
+);
+
+type CampaignPageTypes = Exclude<PageTypes, 'error' | 'home'>;
```

Comment:
`home` will likely be added soon, so it should be a case of removing it from here and adding it into the switch statement below